authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
++
+[NOTE]
+====
+After creating the cluster admin user, xref:../authentication/remove-kubeadmin.adoc#removing-kubeadmin_removing-kubeadmin[delete the existing kubeadmin user] to improve cluster security.
+====
+* xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[Creating service accounts]: Service accounts provide a flexible way to control API access without sharing a regular user’s credentials. A user can xref:../authentication/using-service-accounts-in-applications.adoc#service-accounts-using-credentials-externally_using-service-accounts[use a service account in applications] and also as xref:../authentication/using-service-accounts-as-oauth-client.adoc#using-service-accounts-as-oauth-client[an OAuth client].
+* xref:../authentication/tokens-scoping.adoc#tokens-scoping[Scoping tokens]: A scoped token is a token that identifies as a specific user who can perform only specific operations. You can create scoped tokens to delegate some of your permissions to another user or a service account.
+* xref:../authentication/managing-security-context-constraints.adoc#managing-pod-security-policies[Controlling permissions for pods using security context constraints (SCCs)]: You can use SCCs to define a set of conditions that determine the actions a pod can perform and the resources it can access.
+* Impersonating a system admin user: You can grant cluster administrator permissions to a user by xref:../authentication/impersonating-system-admin.adoc#impersonating-system-admin[impersonating a system admin user].
+* Creating user groups and assigning users to them: You can use groups to manage users, change their permissions, and enhance collaboration. By default, users belong to different xref:../authentication/understanding-authentication.adoc#rbac-groups_understanding-authentication[groups] depending on their authentication status. You can create more groups and assign users to those groups.
+* xref:../authentication/ldap-syncing.adoc#ldap-syncing[Syncing LDAP groups]: You can manage user groups in one place by syncing the groups stored in an LDAP server with the {product-title} user groups.
+* Viewing xref:../authentication/using-rbac.adoc#viewing-local-roles_using-rbac[local] and xref:../authentication/using-rbac.adoc#viewing-cluster-roles_using-rbac[cluster] roles and bindings.
+* Creating xref:../authentication/using-rbac.adoc#creating-local-role_using-rbac[local] and cluster roles, and assigning them to a user or group: {product-title} includes a set of xref:../authentication/using-rbac.adoc#default-roles_using-rbac[default cluster roles]. You can create additional xref:../authentication/using-rbac.adoc#creating-cluster-role_using-rbac[cluster roles] and xref:../authentication/using-rbac.adoc#adding-roles_using-rbac[add them to a user or group].
```

Comment:
isn't this just the `Administrators can manage authorization using the RBAC objects` that's mentioned above?

Reply:
Yes, have removed it.